src/status_im/contexts/wallet/send/input_amount/component_spec.cljs

```diff
@@ -100,60 +100,4 @@
           (.then (fn []
                    (h/is-truthy (h/get-by-label-text :button-one))
                    (h/fire-event :press (h/get-by-label-text :button-one))
-                   (h/was-called on-confirm))))))
-
-  (h/test "Try to fill more than limit"
-    (h/setup-subs sub-mocks)
-    (h/render [input-amount/view
-               {:crypto-decimals 10
-                :limit-crypto    286}])
-
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-2))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-9))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-
-    (-> (h/wait-for #(h/is-truthy (h/get-by-text "$290.00")))
-        (.then (fn []
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-backspace))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-8))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-                 (h/wait-for #(h/get-by-text "$2850.00"))))))
-
-  (h/test "Try to fill more than limit"
-    (h/setup-subs sub-mocks)
-    (h/render [input-amount/view
-               {:crypto-decimals 10
-                :limit-crypto    286
-                :on-confirm      #()}])
-
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-2))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-9))
-    (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-
-    (-> (h/wait-for #(h/get-by-text "$290.00"))
-        (.then (fn []
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-backspace))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-8))
-                 (h/fire-event :press (h/query-by-label-text :keyboard-key-5))
-                 (h/wait-for #(h/get-by-text "$2850.00"))))))
-
-  (h/test "Switch from crypto to fiat and check limit"
```

Comment:
this seems like a valid test case still

Reply:
Again, when the input was more than the limit and a switch was happening, the input was changing to the limit. Now, nothing should happen except for the style changing

Reply:
Done